Library/Formula/wine.rb

```diff
@@ -119,7 +105,7 @@ def wine_wrapper; <<-EOS.undent
   end
 
   def install
-    ENV.m32 # Build 32-bit; Wine doesn't support 64-bit host builds on OS X.
+    ENV.m32 if build.without? "win64"
```

Comment:
We should probably leave this line how it was for now, since the `with-win64` option is a no-op right now on OS X.